src/main/fc/cli.c

```diff
@@ -1759,6 +1759,81 @@ static void cliSerial(char *cmdline)
     memcpy(currentConfig, &portConfig, sizeof(portConfig));
 }
 
+#ifndef SKIP_SERIAL_PASSTHROUGH
```

Comment:
OK,,, Should I update all SKIP_SERIAL_PASSTHROUGH to USE_SERIAL_PASSTHROUGH? I search whole files and found one in line 499 of io/serial.c .